cli/commands/search.py

```diff
@@ -0,0 +1,166 @@
+            query = json.dumps(json.load(f))
+
+        rendered_query = chevron.render(query, {"query": search_terms})
+        response = rank_client.search(
+            index=index,
+            query=json.loads(rendered_query),
+            size=n,
+            source=[
+                "display.referenceNumber",
+                "display.title",
+                "display.production",
+            ],
+        )
+
+        total = response["hits"]["total"]["value"]
+
+        # Use rich tables to display the output
+        table = rich.table.Table(
+            caption=f"Found {total} results",
+            caption_justify="left",
+            show_header=True,
+            show_footer=False,
+            box=rich.box.HEAVY_EDGE,
+            leading=1,
+        )
+        table.add_column("i", justify="right", no_wrap=True)
+        table.add_column("Score", justify="right", no_wrap=True)
+        table.add_column("ID", justify="left", no_wrap=True)
+        table.add_column("Title", justify="left", no_wrap=False)
+        table.add_column("Dates", justify="left", no_wrap=True)
+        table.add_column("Reference number", justify="left", no_wrap=True)
+
+        for i, hit in enumerate(response["hits"]["hits"]):
+            url = f"https://wellcomecollection.org/{content_type}/{hit['_id']}"
+
+            score = str(hit["_score"])
+            title = hit["_source"]["display"]["title"]
+
+            if len(hit["_source"]["display"]["production"]) > 0:
+                dates = ", ".join(
+                    [
+                        date["label"]
+                        for production in hit["_source"]["display"][
+                            "production"
+                        ]
+                        for date in production["dates"]
+                    ]
+                )
+            else:
+                dates = None
+
+            try:
+                reference_number = hit["_source"]["display"]["referenceNumber"]
+            except KeyError:
+                reference_number = None
+
+            table.add_row(
+                str(i + 1),
+                score,
+                f"[link={url}]{hit['_id']}[/link]",
+                f"[link={url}]{title}[/link]",
+                dates,
+                reference_number,
+            )
+
+        rich.print(table)
+
+
+@app.command()
+def get_terms(
+    content_type: Optional[ContentType] = typer.Option(
+        None,
+        help="The content type to find real search terms for",
+        callback=prompt_user_to_choose_a_content_type,
+    ),
+):
+    """Get a list of real search terms for a given content type"""
+    reporting_client = elasticsearch.reporting_client(session)
+    response = reporting_client.search(
+        query={
+            "bool": {
+                "filter": [
+                    {"exists": {"field": "page.query.query"}},
+                    {"term": {"page.name": content_type.value}},
```

Comment:
This equivalence (that `page.name` will be the same as the index name among other things) seems quite risky - I think a more explicit mapping (whether thats a dict or a function or a load of if/elses) between content type and page name would be preferable.